src/openrct2/ride/Track.cpp

```diff
@@ -563,6 +564,24 @@ bool TrackElement::IsBlockStart() const
     return false;
 }
 
+void TrackElement::SetBrakeClosed2(const CoordsXY& trackLocation, bool isClosed)
```

Comment:
Don't make this a member function and rename it so that its clear this is a multi tile set brake.

Reply:
What would be the preference? Passing TrackElement as an argument?

Reply:
Yeah just pass track element by ref as an arg. The main reason I'm not wanting it a member function is that its iterating onto other elements when on multi tile elements. I would expect member functions to only be touching the singular tile.